packages/client/src/hooks/useAccount.js

```diff
@@ -0,0 +1,61 @@
+import { mutate, query, tx } from "@onflow/fcl";
+import { GetAccountBalanceByContractName, SEND_TOKENS_TO_TREASURY } from "flow";
+import { COIN_TYPE_TO_META } from "constants/maps";
+import { COIN_TYPES } from "constants/enums";
+import { REGULAR_LIMIT } from "constants/constants";
+
+const COIN_TYPE_LIST = [COIN_TYPES.FLOW, COIN_TYPES.FUSD, COIN_TYPES.USDC];
+
+export default function useAccount() {
+  const doSendTokensToTreasury = async (treasuryAddr, amount, coinType) => {
+    const vaultPath = COIN_TYPE_TO_META[coinType].storageVaultPath;
+    return await mutate({
+      cadence: SEND_TOKENS_TO_TREASURY,
+      args: (arg, t) => [arg(treasuryAddr, t.Address), arg(amount, t.UFix64), arg(vaultPath, t.Path)],
+      limit: REGULAR_LIMIT,
+    });
+  };
+
+  //FOR TESTING UTILS ONLY, calling this method deposits tokens to treasury
+  //should only be used for testing
+  const initDepositTokensToTreasury = async (treasuryAddr) => {
+    for await (const coinType of COIN_TYPE_LIST) {
```

Comment:
can we move this definition 
`const COIN_TYPE_LIST = [COIN_TYPES.FLOW, COIN_TYPES.FUSD, COIN_TYPES.USDC];`
to keep it inside `initDepositTokensToTreasury`?

Reply:
we use this list in getUserBalances as well